### PR TITLE
June docs

### PR DESCRIPTION
**Pull request by froboy**
- DS-1281 Add AF Wiki [/docs/development/program-event-framework/activity-finder/](https://deploy-preview-88--sparkling-souffle-4f78b0.netlify.app/docs/development/program-event-framework/activity-finder/)
- Massive changes to Activity Finder info in both user docs and dev
  - [AF User docs](https://deploy-preview-88--sparkling-souffle-4f78b0.netlify.app/docs/user-documentation/schedules/activity-finder/)
  - [AF Dev docs](https://deploy-preview-88--sparkling-souffle-4f78b0.netlify.app/docs/development/program-event-framework/activity-finder/) via import. See https://github.com/YCloudYUSA/yusaopeny_activity_finder/pull/43
  - [Traction Rec Mapping](https://deploy-preview-88--sparkling-souffle-4f78b0.netlify.app/docs/development/program-event-framework/traction-rec#mapping) via import. See https://github.com/YCloudYUSA/openy_traction_rec/pull/12

### CODE REVIEW

File: content/en/docs/development/program-event-framework/activity-finder/solr-configuration/_index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+title: Configuring Solr for Activity Finder
+---
+
+In order to install Open Y and Activity Finder v4 you need to run command
+
+```bash
+composer create-project ymcatwincities/openy-project build --no-interaction --prefer-dist
+```
+
+Which will pull Open Y on Drupal 9 stable version with Activity Finder v4 latest stable version
```
Comment (podarok) [inline content/en/docs/development/program-event-framework/activity-finder/solr-configuration/_index.md:11]:
```suggestion
Which will pull Open Y on Drupal stable version with Activity Finder v4 latest stable version
```

File: content/en/docs/development/program-event-framework/activity-finder/solr-configuration/_index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Then you should proceed with a regular installation with Demo content enabled as described in our tutorials. It’s better to setup Extended or Custom( only via drush ) in order to skip a bunch of manual steps
+
+When you have Open Y installed, list of the command you need to run in order to enable Activity Finder v4
```
Comment (podarok) [inline content/en/docs/development/program-event-framework/activity-finder/solr-configuration/_index.md:14]:
```suggestion
When you have YMCA Website Services (former Open Y) installed, list of the command you need to run in order to enable Activity Finder v4
```

**Review by podarok — APPROVED** [review]
TO consider some changes, mentioned above @froboy

**Comment by froboy** [comment]
Thanks for the suggestions @podarok. Added and merging.

**Pull request merged**